tox.ini

```diff
@@ -56,4 +91,4 @@ extras =
 commands=
     py.test check_sphinx.py -v \
     --tb=auto \
-    {posargs}
+    {posargs}
```

Comment:
please add an `.editorconfig` file that enables newlines at the end of a file on save